### PR TITLE
app-crypt/veracrypt: revbump with added patch to fix #605018

### PR DESCRIPTION
**Pull request by soredake**

### CODE REVIEW

**Comment by gktrk** [comment]
As it stands,  this PR depends on https://github.com/gentoo/gentoo/pull/3379 being merged

File: app-crypt/veracrypt/veracrypt-1.19-r1.ebuild [hunk 1]
```diff
@@ -45,6 +45,15 @@ PATCHES=(
 	"${FILESDIR}/${P}-remove-packaging-from-makefile.patch"
 )
 
+src_prepare() {
+	default
+	WX_BASENAME=$( ${EROOT}usr/bin/wx-config --basename )
```
Comment (mgorny) [inline app-crypt/veracrypt/veracrypt-1.19-r1.ebuild]: `${EROOT}` is not available in src_* phases. Just rely on PATH working correctly, and call the executable by name.

File: app-crypt/veracrypt/veracrypt-1.19-r1.ebuild [hunk 2]
```diff
@@ -45,6 +45,15 @@ PATCHES=(
...
+	if [[ ${WX_BASENAME} == "wx_baseu" ]] ; then
```
Comment (mgorny) [inline app-crypt/veracrypt/veracrypt-1.19-r1.ebuild]: Conditional patching is strongly discouraged. Can't you get the patch to be correct unconditionally? Then you should submit it upstream.

**Comment by mgorny** [comment]
#3379 has been rejected due to no activity, and this is being closed for the same reason. Feel free to reopen if you are still interested.

**Pull request closed without merging**